### PR TITLE
Vault Sec buffs

### PR DESCRIPTION
**Pull request by AsterixCodix**
## About The Pull Request

Provides buffs for Vault Sec and renames Overseer to Vault Coordinator (been meaning to for a while).

As it turns out, I pushed to the wrong branch. So... Fixed the Incinerator not firing.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Added Vault-Tec Power Armour
add: Added Vault-Tec Power Armour Helmet
add: Added Vault-Tec Security Specialist
add: Added Riot Suppression Vault-Sec Specialist/ CoS loadout
add: Added Combat Medic Vault-Sec Specialist loadout
add: Added Daily Duty Vault-Sec CoS loadout
add: Added Fire Support Vault-Sec Specialist loadout
add: Added Rat Hunting Vault-Sec loadout
add: Added Policing Vault-Sec loadout
tweak: renamed Overseer to Vault Coordinator

fix: fixed the Incinerator not firing
/:cl:

### CODE REVIEW

**Comment by BadAtThisGame302** [comment]
Erm.. what is a "Vault Coordinator"... (and why did you mean to rename Overseer to this for *awhile* now)

File: code/modules/jobs/job_types/vault.dm [hunk 1]
```diff
@@ -214,15 +235,65 @@ Scientist
...
+
+/datum/job/vault/f13specialistofficer
+	title = "Vault-tec Security Specialist"
+	flag = F13SPECIALISTOFFICER
+	department_head = list("Chief of Security")
+	total_positions = 1
+	spawn_positions = 1
+	forbids = "The Vault forbids: Disobeying the Overseer. Deserting the Vault unless it is rendered unhospitable. Killing fellow Vault Dwellers. Betraying the Vault and its people."
+	enforces = "The Vault expects: Contributing to Vault society. Adherence to Vault-tec Corporate Regulations. Participation in special projects, as ordered by the Overseer."
+	description = "You answer directly to the Chief of Security, and in their absence, the Overseer. You are the first line of defense against civil unrest and outside intrusion. It is your duty to enforce the laws created by the Overseer and proactively seek out potential threats to the safety of Vault residents."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/vault.dm:250]: You changed the Overseer to "Vault Coordinator" and forgot this.

**Comment by AsterixCodix** [comment]
> Erm.. what is a "Vault Coordinator"... (and why did you mean to rename Overseer to this for _awhile_ now)

Gameplay wise, it's the exact same as the Overseer. Trying to steer the Vault in any particular direction is a bit difficult when the Overseer changes every round, if there even is one at all in round. It's essentially like if we let players play as an Elder, a Legate, or an NCR Colonel. We've had the problem before where, we've basically had to push the Enclave envelope a bit in order to steer the Vault in a certain direction, and I'd rather not do that.

**Comment by Zero-Percent-Angel** [comment]
some conflicts on this!

**Comment by AsterixCodix** [comment]
I'll sort the conflicts soon

**Comment by AsterixCodix** [comment]
These conflicts can suck my dick...

**Pull request merged**